.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: release
+on: 
+  push:
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: vmware-tanzu/asset-relocation-tool-for-kubernetes@main
```

Comment:
I don't fully understand what would be the value of using this action. In my head a relok8s action should actually be something like
```
name: relok8s-chart-move
parameters: all the required arguments by the CI
```

and once you add that action + arguments to a workflow a move is performed. This workflow seems to just download the binary on a generic ubuntu image.

maybe, as I asked in another comment, an explanation on how you see this action being used outside of this repository would be useful.

Thanks!